docs/loadflow/loadflow.md

```diff
@@ -91,6 +99,34 @@ $$v_{b_1} + s \cdot q_{svc} = V^{c}_{b_1}$$
 
 Where $s$ is the slope of the static var compensator.
 
+### Computing HVDC power transit
+
+#### LCC converters
+
+TODO
+
+#### VSC converters
+
+VSC converters are self commutated converters that can be assimilated as generators in the loadflow. 
+There can be two main active power regulation mode:
```

Comment:
In general, we say that an HVDC line can be operated in... note that in Powsybl, we don't say "Constant power flow" but with "Active power setpoint". Maybe in not correct for you, but coherence is key for a good user experience.